### PR TITLE
Update dependencies to Polkadot v1.1.0

### PR DESCRIPTION
**Pull request by maltekliemann**

### CODE REVIEW

File: runtime/common/src/lib.rs [hunk 1]
```diff
@@ -548,16 +687,17 @@ macro_rules! impl_config_traits {
...
-            type MinCandidateStk = MinCollatorStk;
-            type MinCollatorStk = MinCollatorStk;
-            type MinDelegation = MinDelegatorStk;
-            type MinDelegatorStk = MinDelegatorStk;
+            type MinCandidateStk = MinCandidateStk;
+            type MinDelegation = MinDelegation;
             type MinSelectedCandidates = MinSelectedCandidates;
             type MonetaryGovernanceOrigin = EnsureRoot<AccountId>;
             type OnCollatorPayout = ();
+            type OnInactiveCollator = OnInactiveCollator;
```
Comment (Chralt98) [inline runtime/common/src/lib.rs:700]:
```rust
pub struct PayoutCollatorOrOrbiterReward;
impl pallet_parachain_staking::PayoutCollatorReward<Runtime> for PayoutCollatorOrOrbiterReward {
	fn payout_collator_reward(
		for_round: pallet_parachain_staking::RoundIndex,
		collator_id: AccountId,
		amount: Balance,
	) -> Weight {
		let extra_weight = if MoonbeamOrbiters::is_orbiter(for_round, collator_id) {
			MoonbeamOrbiters::distribute_rewards(for_round, collator_id, amount)
		} else {
			ParachainStaking::mint_collator_reward(for_round, collator_id, amount)
		};

		<Runtime as frame_system::Config>::DbWeight::get()
			.reads(1)
			.saturating_add(extra_weight)
	}
}

pub struct OnInactiveCollator;
impl pallet_parachain_staking::OnInactiveCollator<Runtime> for OnInactiveCollator {
	fn on_inactive_collator(
		collator_id: AccountId,
		round: pallet_parachain_staking::RoundIndex,
	) -> Result<Weight, DispatchErrorWithPostInfo<PostDispatchInfo>> {
		let extra_weight = if !MoonbeamOrbiters::is_orbiter(round, collator_id.clone()) {
			ParachainStaking::go_offline_inner(collator_id)?;
			<Runtime as pallet_parachain_staking::Config>::WeightInfo::go_offline(
				pallet_parachain_staking::MAX_CANDIDATES,
			)
		} else {
			Weight::zero()
		};

		Ok(<Runtime as frame_system::Config>::DbWeight::get()
			.reads(1)
			.saturating_add(extra_weight))
	}
}
```

This is what I have found for moonbeam. I need to double check if we implemented the correct behaviour.

File: zrml/hybrid-router/src/tests/buy.rs [hunk 2]
```diff
@@ -673,7 +673,7 @@ fn buy_fails_if_balance_too_low() {
         let asset = Assets::CategoricalOutcome(market_id, 0);
         let amount = 10 * BASE;
 
-        assert_ok!(Balances::set_balance(RuntimeOrigin::root(), ALICE, amount - 1, 0));
+        assert_eq!(Balances::set_balance(&ALICE, amount - 1), amount - 1);
```
Comment (Chralt98) [inline zrml/hybrid-router/src/tests/buy.rs:676]: Alice is able to set the balance?

File: zrml/swaps/rpc/src/lib.rs [hunk 3]
```diff
@@ -200,10 +191,14 @@ where
...
         blocks
             .into_iter()
-            .map(|block| {
-                let hash = BlockId::number(block);
+            .map(|block_number| {
+                let hash = self
+                    .client
+                    .hash(block_number)
+                    .unwrap_or(Some(self.client.info().best_hash))
+                    .unwrap_or(self.client.info().best_hash);
```
Comment (Chralt98) [inline zrml/swaps/rpc/src/lib.rs:199]: I need to double-check this.

**Review by Chralt98 — APPROVED** [review]

**Comment by Chralt98** [comment]
Apart from the comments, I approve this. I think the above comments are likely to be ignored.

File: zrml/hybrid-router/src/tests/buy.rs (hunk 2 above)
Comment (maltekliemann) [reply zrml/hybrid-router/src/tests/buy.rs:676]: No, the `ALICE` object is just the address whose balance is set. The `set_balance` call is not publicly exposed API.

**Pull request merged**